website/integrations/documentation/paperless-ngx/index.mdx

```diff
@@ -79,18 +79,33 @@ environment:
             "OAUTH_PKCE_ENABLED": "True"
           }
         }
-    PAPERLESS_AUTO_LOGIN=true
-    PAPERLESS_AUTO_CREATE=true
+    PAPERLESS_AUTO_LOGIN: true
+    PAPERLESS_AUTO_CREATE: true
     PAPERLESS_LOGOUT_REDIRECT_URL: "https://authentik.company/application/o/<application_slug>/end-session/"
 ```
 
+Alternatively, add the variables directly to your environment file:
+
+```sh
+PAPERLESS_ENABLE_ALLAUTH=true
+PAPERLESS_APPS=allauth.socialaccount.providers.openid_connect
+PAPERLESS_SOCIALACCOUNT_PROVIDERS={"openid_connect": {"APPS": [{"provider_id": "authentik", "name": "authentik", "client_id": "<client_id>", "secret": "<client_secret>", "settings": {"server_url": "https://authentik.company/application/o/<application_slug>/.well-known/openid-configuration", "claims": {"username": "email"}}}], "OAUTH_PKCE_ENABLED": "True"}}
+PAPERLESS_AUTO_LOGIN=true
+PAPERLESS_AUTO_CREATE=true
+PAPERLESS_LOGOUT_REDIRECT_URL=https://authentik.company/application/o/<application_slug>/end-session/
+```
+
+To add authentik authentication to an existing Paperless-ngx user, log in to Paperless-ngx with local authentication, click the profile icon in the top-right, click **My Profile**, then **Connect new social account**.
+
+If you want to be able to automatically sign up authentik users, set [PAPERLESS_SOCIAL_AUTO_SIGNUP](https://docs.paperless-ngx.com/configuration/#PAPERLESS_SOCIAL_AUTO_SIGNUP) and/or [PAPERLESS_SOCIALACCOUNT_ALLOW_SIGNUPS](https://docs.paperless-ngx.com/configuration/#PAPERLESS_SOCIALACCOUNT_ALLOW_SIGNUPS).
```

Comment:
Thanks for this contribution @mroetsc ! In this section, do you need to sset those values to `true` or..  what is the exact step to take?